west.yml

```diff
@@ -61,7 +61,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: 962eb7320fdb76ffe265d8bab7330ad472cb6f28
+      revision: pull/1503/head
```

Comment:
Let's give it a chance.
revision: c5c7fb3967905dcaef1035ac8d5b234176a2cc0a
https://github.com/nrfconnect/sdk-zephyr/commit/c5c7fb3967905dcaef1035ac8d5b234176a2cc0a